src/server/api/schema.js

```diff
@@ -632,6 +633,31 @@ const rootMutations = {
 
       return await Organization.get(organizationId);
     },
+    updateTwilioAuth: async (
+      _, {
+        organizationId,
+        twilioApiKey,
+        twilioAuthToken,
+        twilioMessageServiceSid
+      }, {
+        user
+      }
+    ) => {
+      await accessRequired(user, organizationId, "OWNER");
+
+      const organization = await Organization.get(organizationId);
+      const featuresJSON = JSON.parse(organization.features || "{}");
+      featuresJSON.TWILIO_API_KEY = twilioApiKey;
```

Comment:
for all of these values, can you accept a max-length.  i.e. `featuresJSON.TWILIO_AUTH_TOKEN = twilioAuthToken.substr(0, 1024)`

Reply:
I put a max length on the frontend. That work?
e.g. `apiKey: yup.string().nullable().max(64)` in Settings.jsx

Reply:
right -- let's ALSO have a max on the backend.  64 is even a better max -- it both communicates in the backend code what is expected and also stops some (admittedly obscure, but nonetheless) vulnerabilities.